### PR TITLE
Abstract

### PR DESCRIPTION
**Pull request by nillas12**
### Description
Abstract

### Changelog
Abstrcat

### Issues solved 
add these with the respective key as HCLANG-1 (as an example)
 - HCLANG-141

### Issues opened as a result of this 
Add something here if new issues arise because of this. For instance if new features-requests or bugs arise from this PR. If this is not the case, then remove this section.

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding filenames
- all .tex files use pascal case
- all other files [except in /code/] use snake case

### Regarding code [if applicable]
- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
- Are public methods documented? 
- Are there testcases for new features / methods?
- Have you tried running the source and tested things that cannot be test-cased?

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

File: report/Titelblad.tex [hunk 1]
```diff
@@ -33,7 +33,16 @@
   \includegraphics[width=0.9\textwidth]{images/aau_logo.pdf}\\[1.0 cm]
  \end{flushright}
  \begin{flushleft}
-  \textbf{Abstract:}\\INSERT HERE
+  \textbf{Abstract:}\\
+  In the danish education system, the Arduino platform is often used to introduce the students to computer science.
```
Comment (jones1994) [inline report/Titelblad.tex]: Remove "the" before "students"

File: report/Titelblad.tex [hunk 2]
```diff
@@ -33,7 +33,16 @@
   \includegraphics[width=0.9\textwidth]{images/aau_logo.pdf}\\[1.0 cm]
  \end{flushright}
  \begin{flushleft}
-  \textbf{Abstract:}\\INSERT HERE
+  \textbf{Abstract:}\\
+  In the danish education system, the Arduino platform is often used to introduce the students to computer science. 
+  The Arduino's hardware is relatively simple simple and easy to grasp, however the Arduino language can often be difficult for beginners.
```
Comment (jones1994) [inline report/Titelblad.tex]: Remove one of the "simple"

Comment (jones1994) [reply report/Titelblad.tex]: Also Change "beginners" -> "novices"

File: report/Titelblad.tex [hunk 3]
```diff
@@ -33,7 +33,16 @@
...
+  The Arduino language is a subset of C++, and while it is simplified compared to C++, there are still parts of the language that are difficult to comprehend for novices.
```
Comment (Abildgaard96) [inline report/Titelblad.tex]: maybe change to 
While the Arduino language is a simplified subset of C++, there are still ....
just to shorten a bit.

File: report/Titelblad.tex (hunk 1 above)
Comment (Abildgaard96) [inline report/Titelblad.tex]: danish -> Danish
also i prefer educational system, compared to education system.

File: report/Titelblad.tex (hunk 2 above)
Comment (Abildgaard96) [inline report/Titelblad.tex]: double 'simple'. 
should also be.
...to grasp.
However, the...

File: report/Titelblad.tex [hunk 4]
```diff
@@ -33,7 +33,16 @@
...
+  
+  This report is meant to elaborate and explain the development of the HCL language, a programming language designed to ease the introduction to programming for students and novices.
```
Comment (Abildgaard96) [inline report/Titelblad.tex]: reformulate to:
This report elaborates and explains...
meant is the alias of will

File: report/Titelblad.tex [hunk 5]
```diff
@@ -33,7 +33,16 @@
...
+  HCL does this by having an emphasis on making the language read like english.
```
Comment (Abildgaard96) [inline report/Titelblad.tex]: read like english -> resemble the English language.

**Pull request merged**